flake.nix

```diff
@@ -1,6 +1,6 @@
 {
   inputs = {
-    nixpkgs.url = "github:nixos/nixpkgs/nixos-22.05";
+    nixpkgs.url = "github:nixos/nixpkgs/haskell-updates";
```

Comment:
```suggestion
    nixpkgs.url = "github:nixos/nixpkgs/nixpkgs-unstable";
```

Reply:
I used master instead; thanks for the catch on using a temporary branch.